### PR TITLE
add bootstrap governance doc

### PR DESCRIPTION
**Pull request by sbtaylor15**

### CODE REVIEW

**Review by steveonjava — APPROVED** [review]

File: governance.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+* Slack: #pyrsia_bootstrap [slack channel](https://openssf.slack.com/archives/C02RC7Y5EUV) on the OpenSSF slack.
+
+Governance decisions, votes and questions should take place on the pyrsia@googlegroups.com mailing list.
+
+## Permissions and access
+
+Members of the governing board will be given access to these resources:
+
+* Google Groups Administrators
+* [GitHub Org](https://github.com/pyrsia) Administrators
```
Comment (prince-chrismc) [inline governance.md:104]: I'll need to add more people 😉 


Maybe a more precise link? 

```suggestion
* [GitHub Org](https://github.com/orgs/pyrsia/teams/admins) Administrators
```

**Review by prince-chrismc — COMMENTED** [review]
Couple nits but overwhelming good

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+* Document process responsible parties go through to triage and determine veracity of vulnerability
+* Document process for facilitating fix, generating release update, and communicating vulnerability and fix to public
+
+## Code of Conduct
+
+The code of conduct MUST set expectations for contributors on expected behavior, as well as explaining the consequences of violating the terms of the code.
+The [Contributor Covenant](https://www.contributor-covenant.org) has become the de facto standard for this language.
+
+Members of the governance committee will be responsible for handling [Pyrsia code of conduct](code-of-conduct.md)
+violations via pyrsia-code-of-conduct@googlegroups.com.
```
[the remark this answers is not in the document]
Comment (prince-chrismc) [reply governance.md:87]: https://github.com/pyrsia/.github/blob/main/code-of-conduct.md#enforcement

**Review by mgrand — APPROVED** [review]

File: governance.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sbtaylor15) [reply governance.md]: Changed to use openssf email

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
There's no merge rules on this repo, so feel free to merge whenever your think it's ready!

(thought you now have three ✅ )

**Pull request merged**